control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -335,8 +340,14 @@ func buildCVOContainerMain(image, releaseImage string) func(c *corev1.Container)
 			fmt.Sprintf("--listen=0.0.0.0:%d", port),
 			fmt.Sprintf("--serving-cert-file=%s", cpath(cvoVolumeServerCert().Name, corev1.TLSCertKey)),
 			fmt.Sprintf("--serving-key-file=%s", cpath(cvoVolumeServerCert().Name, corev1.TLSPrivateKeyKey)),
+			"--hypershift=true",
```

Comment:
This line is fine, just thinking it through out loud in case other folks are thinking in the same direction.  CVO support for this option is new in 4.15:

```console
cluster-version-operator $ git --no-pager grep hypershift origin/release-4.15
origin/release-4.15:cmd/start.go:       cmd.PersistentFlags().BoolVar(&opts.InjectClusterIdIntoPromQL, "hypershift", opts.InjectClusterIdIntoPromQL, "This options indicates whether the CVO is running inside a hosted control plane.")
cluster-version-operator $ git --no-pager grep hypershift origin/release-4.14
...no hits...
```

But this HostedControlPlane operator is pinned to the release payload, so we should be able to set it like you have it without worrying about version skew vs. the CVO.  Older payloads with CVOs that don't support this option will have older HostedControlPlane controllers that don't try to set it.